mitmproxy/contentviews/msgpack.py

```diff
@@ -48,3 +48,6 @@ def __call__(self, data, **metadata):
         data = parse_msgpack(data)
         if data is not PARSE_ERROR:
             return "MsgPack", format_msgpack(data)
+
+    def should_render(self, content_type):
+        return content_type in self.__content_types
```

Comment:
Not inlining probably reduces the number of allocations, but this is not a hot path and we shouldn't care. Either option is fine.

More practically speaking, we should consider keeping an documented `content_types` ClassVar and have a default `View.should_render` implementation that checks just that. This way we could retain backward-compatibility with third-party views.

Reply:
Well, I thought it could be misleading to have both `content_types` and `should_render`. `should_render` could do more than `content_types` does. Keeping `content_types` visible would encourage 3rd party users to still used, which could not be appropriate all the time. But the argument about keeping 3rd party views worries me and I am not sure how to handle it.

So, do you think we should have both `content_types` and `should_render` visible even when they are potentially incompatible?